File: NEFBDAACommons/Shared/Models/AppLogViewModel.cs
namespace NEFBDAACommons.Shared.Models
{
    public class AppLogViewModel : BaseViewModel<long>
    {
        public long? CreatorUserId { get; set; }
        public string Level { get; set; }
        public string Timestamp { get; set; }
        public string FileName { get; set; }
        public string LineNumber { get; set; }
        public string Message { get; set; }
        public object Additional { get; set; }

        public override string OptionText => $"{Level} {Timestamp} {FileName} {LineNumber} {Message}";
    }
}

File: src/NEFBDAAStarter/angular/src/app/generated-api/models/app-log-view-model.ts
/* tslint:disable */
export interface AppLogViewModel {
  creatorUserId?: number;
  level?: string;
  timestamp?: string;
  fileName?: string;
  lineNumber?: string;
  message?: string;
  additional?: {};
  optionText?: string;
  id?: number;
  guid?: string;
  optionId?: string;
  isDeleted?: boolean;
  optionAdditionalInfo?: {[key: string]: string};
  hideFromFilter?: boolean;
  hideFromEdit?: boolean;
}
